roles/sap_hana_rhv_guest/tasks/set-tuned-profile.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Create tuned profile directory /usr/lib/tuned/sap-hana
+  file:
+    path: /usr/lib/tuned/sap-hana
+    state: directory
+
+- name: Create sap-hana tuned profile
+  copy:
+    dest: "/usr/lib/tuned/sap-hana/tuned.conf"
+    content: |
+        #
+        # tuned configuration
+        #
+        [main]
+        summary=Optimize for SAP HANA
+
+        [cpu]
+        force_latency=cstate.id:3|70
+        governor=performance
+        energy_perf_bias=performance
+        min_perf_pct=100
+
+        [vm]
+        transparent_hugepages=never
+
+        [sysctl]
+        kernel.sem = 32000 1024000000 500 32000
+        kernel.numa_balancing = 0
+        kernel.sched_min_granularity_ns = 3000000
+        kernel.sched_wakeup_granularity_ns = 4000000
+        vm.dirty_ratio = 40
+        vm.dirty_background_ratio = 10
+        vm.swappiness = 10
+
+
+- name: Create tuned profile directory /usr/lib/tuned/sap-hana-kvm-guest
+  file:
+    path: /usr/lib/tuned/sap-hana-kvm-guest
+    state: directory
+
+- name: Add haltpoll.sh for tuned sap-hana-kvm-guest
```

Comment:
Again why use content when this file is available in the role. 
Let's keep only one.